frontend/src/metabase/common/components/DashboardPicker/components/DashboardPickerModal.tsx

```diff
@@ -223,4 +223,12 @@ export const DashboardPickerModal = ({
       />
     </>
   );
+
+  const modalCtx = useModalStackContext();
```

Comment:
This is really annoying, but since we know that there can be other modals shown here, we need to ensure that these have access to a modal stack (if they don't already). I'm not sure what happens when you nest modal stacks... but I didn't really want to find out either haha

Reply:
This logic ends up used in the "move question" and "Add to dashboard" type flows